slackclient/server.py

```diff
@@ -193,7 +199,7 @@ def websocket_safe_read(self):
             try:
                 data += "{0}\n".format(self.websocket.recv())
             except SSLError as e:
-                if e.errno == 2:
+                if e.errno == 2 or e.errno == 11:
```

Comment:
We'll need to add comments about when `errno 2` occurs to the comments below

Reply:
ok.

Reply:
Actually, I just took out the or e.errno == 11

I couldn't find anything on the net to justify it, and when I was getting it, my (outside of the client code) was fairly unhappy; 40K unsent messages, spawning up thousands of threads to deliver them all, paying no attention to the rate limiting errors or server closed socket unexpectedly etc.  

The main other change I made was outside of slackclient code, but possibly should be in it, or at least documented as good for large installations:  serialize all send to slack message sending; Now my "send_to_slack" function just appends the msg to a list, and there is a greenlet dequeuing them, honoring explicit rate limits, and sleeping 2 minutes when I get a sudden unexpectd PIPE error due to remote closing socket.